task_8/Aleksandr.Timoshenko/src/main/java/com/hillel/fleshlight/Flashlight.java

```diff
@@ -0,0 +1,44 @@
+package com.hillel.fleshlight;
+
+public class Flashlight {
+
+  public static final int MAX_BATTERY = 3;
+
+  private int battery = 0;
+  private String light;
+  private boolean shine;
+
+
+  public String lightType () {
+    switch (battery) {
+      case 0:
```

Comment:
Use constants here instead of 0, 1, 2, 3, the best options to use enum construct